utils/utils.js

```diff
@@ -327,6 +327,57 @@ function findAllEnabledDestinations(sdkSuppliedIntegrations, configPlaneEnabledI
   }
 
 }
+function rejectArr(obj, fn) {
+  fn = fn || compact;
+  return "array" == type(obj) ? rejectarray(obj, fn) : rejectobject(obj, fn);
+}
+
+var rejectarray = function (arr, fn) {
```

Comment:
add method description about the functionality

Reply:
done